utils/cli-utils/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "cli-utils"
```

I am afraid the name [`cli-utils` is already occupied on crates.io](https://crates.io/crates/cli-utils). But even if not, these are binding frame and substrate so the name and location should reflect that:
```suggestion
name = "substrate-frame-cli-utils"
```

And the crate should be located at `utils/frame/cli-utils`.